docs/overview.md

```diff
@@ -461,6 +461,416 @@ Webvis video at `http://localhost:8000/yet_another`.
+set OPENLINEAGE_API_KEY=your_api_key
+set OPENLINEAGE_ENDPOINT=/api/v1/lineage
+set OPENLINEAGE_VERIFY_CLIENT_URL=false
+set WEBVIS_MODEL_NAME=my-model-v1
+```
+> **Note:** These variables for Windows will only persist for the current CMD session.
+
+## 📊 Visualizing Events in Marquez
+
+`OpenFilterLineage` emits OpenLineage-compliant events that can be visualized using **Marquez**, an open-source data lineage backend with a web-based UI. Marquez can be used locally as a testing and observability tool, allowing developers to inspect and validate lineage events generated by OpenFilter pipelines in real time.
+
+
+---
+
+### 🚀 Launching Marquez via Docker
+
+To run a local instance of Marquez for testing:
+
+```bash
+git clone https://github.com/MarquezProject/marquez.git
+cd marquez
+./docker/up.sh --seed
+```
+
+> 💡 Use `--seed` to preload demo data.  
+> To start with a clean instance:
+
+```bash
+./docker/up.sh --no-seed
+```
+
+---
+
+### 🌐 Default Service Endpoints
+
+| Component   | URL                                     |
+|-------------|------------------------------------------|
+| Web UI      | http://localhost:3000                   |
+| REST API    | http://localhost:5000                   |
+| Healthcheck | http://localhost:5001/healthcheck       |
+
+---
+
+### 🔗 Connecting OpenFilter to Marquez
+
+Set the following environment variables in your `.env` file or export them in your terminal:
+
+```env
+OPENLINEAGE_URL=http://localhost:5000
+OPENLINEAGE_API_KEY=                     # Optional: leave empty if not required
+OPENLINEAGE_VERIFY_CLIENT_URL=false
+OPENLINEAGE_ENDPOINT=/api/v1/lineage
+OPENLINEAGE_PRODUCER=https://github.com/PlainsightAI/openfilter/tree/0.1.2/openfilter/lineage
+OPENLINEAGE__HEART__BEAT__INTERVAL=10
+
+```
+
+These values are used by the `get_http_client()` method inside `OpenFilterLineage` to configure the OpenLineage HTTP client.
+
+---
+
+### 📈 Accessing and Exploring Events
+
+Once your pipeline is emitting events, open the Marquez UI:
+
+[http://localhost:3000](http://localhost:3000)
+
+You’ll be able to:
+
+- **Browse Jobs** – Each filter is registered as a job (`filter_name`)
+- **Inspect Runs** – Executions with unique `runId`
+- **View Facets** – Metadata like model name, frame rate, etc.
+- **Explore the Lineage Graph** – Dependencies between jobs and datasets
+
+---
+
+### ✅ Example Facet Output in Marquez
+
+```json
+{
+  "eventTime": "2025-06-24 11:19:40.869547000",
+  "eventType": "RUNNING",
+  "inputs": [],
+  "job": {
+    "name": "FilterCrop",
+    "namespace": "Openfilter"
+  },
+  "outputs": [],
+  "producer": "https://github.com/PlainsightAI/openfilter/tree/0.1.2/openfilter/lineage",
+  "run": {
+    "facets": {
+      "openfilter": {
+        "_producer": "https://github.com/PlainsightAI/openfilter/tree/0.1.2/openfilter/lineage",
+        "_schemaURL": "https://github.com/PlainsightAI/openfilter/lineage/schema/OpenFilterConfigRunFacet.json",
+        "_type": "FilterCrop",
```

Comment:
In your `OpenFilterConfigRunFacet` facet, the filter type is `type`, not `_type`. I'd suggested not using the underscore (`_`) as that's a reserved naming convention for OpenLineage to avoid naming conflicts.